.github/workflows/main.yml

```diff
@@ -50,6 +50,15 @@ jobs:
           version: 0.3.1
           args: check --select I
 
+      - name: Check docstrings using interrogate
```

Comment:
we're using interrogate to force ourselves to write docstrings for every function and module